modules/nw-enabling-multicast.adoc

```diff
@@ -24,15 +24,111 @@ You can enable multicast between Pods for your project.
 
 .Procedure
 
-* Run the following command to enable multicast for a project:
+* Run the following command to enable multicast for a project. Replace `<namespace>` with the namespace for the project you want to enable multicast for.
 +
 [subs="attributes+"]
 ----
-$ oc annotate {namespace} <namespace> \ <1>
+$ oc annotate {namespace} <namespace> \
     {annotation}
 ----
+
+.Verification steps
+
+To verify that multicast is enabled for a project, complete the following procedure:
+
+. Change your current project to the project that you enabled multicast for. Replace `<project>` with the project name.
++
+[source,terminal]
+----
+$ oc project <project>
+----
+
+. Create a Pod to act as a multicast receiver:
++
+[source,terminal]
+----
+$ cat <<EOF| oc create -f -
+apiVersion: v1
+kind: Pod
+metadata:
+  name: mlistener
+  labels:
+    app: multicast-verify
+spec:
+  containers:
+    - name: mlistener
+      image: registry.access.redhat.com/ubi8
+      command: ["/bin/sh", "-c"]
+      args:
+        ["dnf install socat hostname && sleep inf"]
+      ports:
+        - containerPort: 30102
+          name: mlistener
+          protocol: UDP
+EOF
+----
+
+. Create a Pod to act as a multicast sender:
 +
-<1> The `namespace` for the project you want to enable multicast for.
+[source,terminal]
+----
+$ cat <<EOF| oc create -f -
+apiVersion: v1
+kind: Pod
+metadata:
+  name: msender
+  labels:
+    app: multicast-verify
+spec:
+  containers:
+    - name: msender
+      image: registry.access.redhat.com/ubi8
+      command: ["/bin/sh", "-c"]
+      args:
+        ["dnf install socat && sleep inf"]
+EOF
+----
+
+. Start the multicast listener:
+
+.. Get the IP address for the Pod:
++
+[source,terminal]
+----
+$ POD_IP=$(oc get pods mserver -o jsonpath='{.status.podIP}')
+----
+
+.. To start the multicast listener, in a new terminal window or tab, enter the following command:
++
+[source,terminal]
+----
+$ oc exec mlistener -i -t -- \
+    socat UDP4-RECVFROM:30102,ip-add-membership=224.1.0.1:$POD_IP,fork EXEC:hostname
+----
+
+. Start the multicast transmitter.
+
+.. Get the Pod network IP address range:
++
+[source,terminal]
+----
+$ CIDR=$(oc get Network.config.openshift.io cluster \
+    -o jsonpath='{.status.clusterNetwork[0].cidr}')
+----
+
+.. To send a multicast message, enter the following command:
++
+[source,terminal]
+----
+$ oc exec mclient -i -t -- \
```

Comment:
Need update to use: oc exec msender -i -t --     /bin/bash -c "echo | socat STDIO UDP4-DATAGRAM:224.1.0.1:30102,range=$CIDR"